src/patternfly/components/Page/page-sidebar-body.hbs

```diff
@@ -0,0 +1,6 @@
+<div class="pf-c-page__sidebar-body{{#if page-sidebar-body--modifier}} {{page-sidebar-body--modifier}}{{/if}}" 
```

Comment:
Looks like you need to update the dataTable and dataList demos with this change, too. How about just adding this to `page-sidebar.hbs`, so updating `page-sidebar.hbs` to...


```
<div class="pf-c-page__sidebar{{#if page-sidebar--modifier}} {{page-sidebar--modifier}}{{/if}}" 
  {{#if page-sidebar--attribute}}
    {{{page-sidebar--attribute}}}
  {{/if}}>
  {{#> page-sidebar-body}}
    {{> @partial-block}}
  {{/page-sidebar-body}}
</div>
```

You could do that to the rest of these changes (accordion and datalist), since the `-body` element is required in each of these components.

Reply:
I added it to accordion, but not datalist because although its required, we need the option to add a modifier to remove padding.